lib/multicell_location/services/location_service.c

```diff
@@ -15,7 +15,12 @@
 #include "skyhook_integration.h"
 #include "polte_integration.h"
 
+#if defined(CONFIG_MULTICELL_LOCATION_SERVICE_NRF_CLOUD) ||                                        \
+	defined(CONFIG_MULTICELL_LOCATION_SERVICE_HERE) ||                                         \
+	defined(CONFIG_MULTICELL_LOCATION_SERVICE_SKYHOOK) ||                                      \
+	defined(CONFIG_MULTICELL_LOCATION_SERVICE_POLTE)
```

Comment:
Why do we need this?
There is BIULD_ASSERT in multicell_location.c which we should hit if this is not true so I won't compile anyway.

Reply:
It looks like `multicell_location.c` and `location_service.c` are built in parallel. `multicell_location.c` hits the build assert (because no location service is enabled in the configuration), but `location_service.c` doesn't have one. If we want to have something in `location_service.c` as well, it should be a similar build assert as in `multicell_location.c`, I think.

Reply:
That's right, I dropped that commit